lib/moirai/engine.rb

```diff
@@ -22,12 +18,18 @@ module ActionView::Helpers::TranslationHelper # rubocop:disable Lint/ConstantDef
           def translate(key, **options)
             value = original_translate(key, **options)
 
+            missing_translationf = value.include?('class="translation_missing"')
```

Comment:
Importatn, otherwise we have the 
missing translation span
https://api.rubyonrails.org/classes/ActionView/Helpers/TranslationHelper.html